benches/benches/block_target_gas.rs

```diff
@@ -239,8 +238,568 @@ fn block_target_gas(c: &mut Criterion) {
     //     .to_vec(),
     // );
 
+    run_alu(&mut group);
+
     group.finish();
 }
 
+fn run_alu(group: &mut BenchmarkGroup<WallTime>) {
+    run(
+        "add opcode",
```

Comment:
It would be nice to add an `alu/` prefix to each benchmark. In this case, we can easily run only all `alu` benchmarks later.